plugin/pkg/admission/serviceplan/defaultserviceplan/admission.go

```diff
@@ -85,7 +78,7 @@ func (d *defaultServicePlan) Admit(a admission.Attributes) error {
 	}
```

Comment:
This should also check if instance.Spec.ClusterServicePlanName != ""
so: 
if  instance.Spec.ExternalClusterServicePlanName != "" || instance.Spec.ClusterServicePlanName != "" {

See PR:
https://github.com/kubernetes-incubator/service-catalog/pull/1377

Perhaps this got lost in rebases?

Reply:
not lost, this PR just needed to be rebased.  Doing it now.

Reply:
Odd, I was expecting the little box on the bottom to say that it needed to be rebased but it didn't. Thanks!!